soul-admin/src/main/resources/static/index.html

```diff
@@ -8,11 +8,11 @@
   <title>soul网关</title>
   <link rel="icon" href="/favicon.png" type="image/x-icon">
   <script src="https://gw.alipayobjects.com/os/antv/assets/data-set/0.8.7/data-set.min.js"></script>
-<link href="/index.2aec5dfe.css" rel="stylesheet"></head>
+<link href="/index.65957209.css" rel="stylesheet"></head>
 
 <body>
   <div id="httpPath" style="display: none">http://localhost:9095</div>
   <div id="root"></div>
-<script type="text/javascript" src="/index.a62f1466.js"></script></body>
+<script type="text/javascript" src="/index.9c985f9d.js"></script></body>
```

Comment:
why not this
?

Reply:
My bad, I change back to the original  value.